src/components/annotations/AnnotationList.js

```diff
@@ -1,8 +1,8 @@
 import React, { Component } from "react";
```

Comment:
Move all the annotation files into 'annotations' folder.
I didn't change any code. Just updated the import path.